monai/data/dataset.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from monai.utils.module import export
+
+
+@export("monai.data")
+class Dataset(torch.utils.data.Dataset):
+    """
+    General Dataset to handle dictionary format data, it can operate transforms for specific fields.
+    For example, typical input data can be a list of dictionaries:
+    [{                            {                            {
+         'img': 'image1.nii.gz',      'img': 'image2.nii.gz',      'img': 'image3.nii.gz',
```

Comment:
Does it make sense to ask user to replicate the 'img' key. I expect most users will have data in format {'img': images, 'label': labels}. @wyli @ericspod can you share your thoughts?